### ISSUE TITLE
How to specify custom week endpoints in apply.weekly of tq_transmute

### ISSUE THREAD

**Issue by McCompexpert** [business-science/tidyquant#104]
for example Monday - Friday, so the week's endpoint should be always Friday?

**Comment by DavisVaughan** [comment]
I don't think you can do this easily, but you can probably use `period.apply()` with a custom INDEX that corresponds to weeks that end on Fridays. https://www.rdocumentation.org/packages/xts/versions/0.10-2/topics/period.apply
